dts/bindings/net/wireless/generic-coex-gpio.yaml

```diff
@@ -0,0 +1,18 @@
+description: |
+    Generic representation of Coexistance pinx for radio
+
+compatible: "generic-coex-gpio"
```

Comment:
how generic is this? from this description I can't understand why.

Reply:
I have updated the description with much longer and hopefully comprehensive one.